packages/components/pie-button/test/accessibility/pie-button.spec.ts

```diff
@@ -1,36 +1,40 @@
 import { test, expect } from '@sand4rt/experimental-ct-web';
-import { PieButton } from '@/index';
-import { BUTTON_SIZE, BUTTON_VARIANT } from '@/defs';
 import AxeBuilder from '@axe-core/playwright';
+import {
+    PropObject, Combination, getAllPropCombinations, splitCombinationsByPropertyValue,
+} from '@justeattakeaway/pie-webc-core/src/test-helpers/get-all-prop-combos.ts';
+import { PieButton } from '@/index';
+import { BUTTON_TYPE, BUTTON_SIZE, BUTTON_VARIANT } from '@/defs';
 
-const sizes = Object.values(BUTTON_SIZE);
-const variants = Object.values(BUTTON_VARIANT);
-const disabledStates = [true, false];
+const props: PropObject = {
+    variant: Object.values(BUTTON_VARIANT),
+    size: Object.values(BUTTON_SIZE),
+    type: BUTTON_TYPE.BUTTON,
```

Comment:
I think a comment might add something here
```suggestion
    type: BUTTON_TYPE.BUTTON, // Changing the type does not affect the appearance of the button
```